src/main/java/uk/org/tombolo/exporter/CSVExporter.java

```diff
@@ -69,8 +69,12 @@ public void write(Writer writer, List<Subject> subjects, List<Field> fields, Boo
 			try {
 				property.put(field.getLabel(), ((SingleValueField) field).valueForSubject(subject, timeStamp));
 			} catch (IncomputableFieldException e) {
-				log.warn("Could not compute Field {} for Subject {}, reason: {}", field.getLabel(), subject.getLabel(), e.getMessage());
+				log.warn("\u001b[0;33m" + "Could not compute Field {} for Subject {}, reason: {}" + "\u001b[m",
+						field.getLabel(), subject.getLabel(), e.getMessage());
```

Comment:
It would be good if we could have a constant class containing these formats, so that we dont have to write them again and again when required, we could just refer to those constants.